sycl/unittests/Extensions/DefaultContext.cpp

```diff
@@ -18,12 +18,7 @@
 inline constexpr auto EnableDefaultContextsName =
     "SYCL_ENABLE_DEFAULT_CONTEXTS";
 
-TEST(DefaultContextTest, DefaultContextTest) {
-  using namespace sycl::detail;
-  using namespace sycl::unittest;
-  ScopedEnvVar var(EnableDefaultContextsName, "1",
-                   SYCLConfig<SYCL_ENABLE_DEFAULT_CONTEXTS>::reset);
-
+void test_contexts_are_equal() {
```

Comment:
```suggestion
void test_default_context_enabled() {
```

Reply:
Previously the function had a different signature, so previous suited better. But now I agree with you, fixed.